### PR TITLE
feat: optional description in TxModal

### PR DESCRIPTION
**Pull request by mhagel**
## Description

Closes: regen-network/regen-registry#1094

- Adds an optional description to the TxModal

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed
- [ ] once the PR is closed, set up backport PRs for `redwood`, `hambach` and optionally `master` (see below)

#### Setting up backport PRs

After merging your PR to `dev`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR:

- If PR is a new feature: `@Mergifyio backport redwood hambach`.
- If PR is a bug fix: `@Mergifyio backport redwood hambach master`.

2. If your branch does have merge commits:

   a. Pull latest `dev`, `hambach` and `redwood` branches

   b. Create new branches for backports and merge `dev` (replace `<PR#>` with your PR #)

   ```
   git checkout -b hambach-backport-<PR#> hambach
   git merge dev
   git push origin hambach-backport-<PR#>
   git checkout -b redwood-backport-<PR#> redwood
   git merge dev
   git push origin redwood-backport-<PR#>`
   ```

   c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.

   d. If PR is a bug fix repeat the same process for `master` branch.

### How to test

1. https://deploy-preview-1517--regen-storybook.netlify.app/?path=/story/modal--tx-successful-modal now includes an example of a description with a link that opens a new tab
Screen Shot 2022-10-25 at 11 36 38 AM

This is the only element I see as necessary to creating this success modal (mock):

Screen Shot 2022-10-25 at 11 38 19 AM

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by erikalogie** [comment]
Ok, looks good to me!

**Review by flagrede — APPROVED** [review]
Looks good!

File: web-components/src/components/modal/TxModal.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import ReactHtmlParser from 'react-html-parser';
```
Comment (flagrede) [inline web-components/src/components/modal/TxModal.tsx:2]: I had to replace this dependency (which is no longer maintained) with `html-react-parser` during React 18 migration.
Could you also replace it in this PR to anticipate the migration merge?

Comment (mhagel) [reply web-components/src/components/modal/TxModal.tsx]: done

**Comment by mhagel** [comment]
@mergifyio backport redwood hambach

**Pull request merged**